experimental/LieAlgebras/src/RootSystem.jl

```diff
@@ -313,6 +514,10 @@ function Base.iszero(r::RootSpaceElem)
   return iszero(r.vec)
 end
 
+function Base.length(r::RootSpaceElem)
+  return dot(r, r)
```

Comment:
We need `sqrt` here, but does this work with current types ?
```suggestion
  return sqrt(dot(r, r))
```

Reply:
No, that doesn't work

Reply:
We could instead name the function something like `length_squared` instead

Reply:
That is neither intuitive nor shorter than `dot(r, r)`, so I don't feel great about this.

Reply:
The let's just remove `length`